quickstart/ia32-generic-qemu.md

```diff
@@ -12,6 +12,7 @@ See [how to build the Phoenix-RTOS system image](../building/README.md).
 Firstly, you need to install QEMU emulator.
   <details>
   <summary>How to get QEMU (Ubuntu)</summary>
+&nbsp;
```

Comment:
To be honest it looks good without a newline, when there is a list within `details`:
![Screenshot from 2023-11-13 12-52-15](https://github.com/phoenix-rtos/phoenix-rtos-doc/assets/77304431/a115d485-1908-41f4-a91f-f9e57279f900)

However I see the another issue, which is:
![Screenshot from 2023-11-13 12-54-56](https://github.com/phoenix-rtos/phoenix-rtos-doc/assets/77304431/c0e32ddf-7487-461f-bc27-47d699753bf4)

From what I checked removing spaces before `<details>` helps